TGC/localisation/00_map-regions.csv

```diff
@@ -6,7 +6,7 @@ CANAL_3;Panama Canal;;;;;;;;;;;;;x
 CANAL_4;Bosphorus Strait;;;;;;;;;;;;;x
 CANAL_5;Dardanelles Strait;;;;;;;;;;;;;x
 ENG_6;MacKenzie District;;;;;;;;;;;;;x
-CANAL_6;English Channel;;;;;;;;;;;;;x
```

Comment:
are you sure this won't break something?

Reply:
Well it's just localization so nope. Canal 6 is never activated anywhere in the game. Presumably someone added this is with the idea that they could add a feature in the future where controlling the land around the English channel might allow them to block it.

Reply:
ah, to probably just a leftover from somewhere? okay